bin/cut_k3s_release_issue

```diff
@@ -0,0 +1,86 @@
+#!/bin/sh
+
+set -e
+
+. ./stdlib-sh/stdlib.sh
```

Comment:
Would there be any added value in utilizing the logger library for error handling>

Reply:
Maybe in a second pass but not overly beneficial at the moment.